makejdk.sh

```diff
@@ -29,7 +29,8 @@ SCRIPT_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
 # shellcheck source=sbin/common-functions.sh
 source "$SCRIPT_DIR/sbin/common-functions.sh"
 
-REPOSITORY=${REPOSITORY:-adoptopenjdk/openjdk-jdk8u}
+#REPOSITORY=${REPOSITORY:-adoptopenjdk/openjdk-jdk8u}
```

Comment:
`jdk8` and `jdk9` seem to be common factors all across, why not extract them into an environment variabel `JDK_VERSION`, that can be reused across the script without having to modify it